Zend/zend_types.h

```diff
@@ -770,13 +770,13 @@ static zend_always_inline uint32_t zval_gc_info(uint32_t gc_type_info) {
 			(b) ? IS_TRUE : IS_FALSE;	\
 	} while (0)
 
-#define ZVAL_LONG(z, l) {				\
+#define ZVAL_LONG(z, l) do {			\
```

Comment:
missing while ...

if these are going to be changed, they should probably all be changed ...

I'm not convinced these need changing ...

Reply:
so sorry, a stupid mistake...

> if these are going to be changed, they should probably all be changed ...

what other macros are not using `do...while`?

Reply:
ah, my mistake :)